### PR TITLE
Added information to help newcomers get allowed

### PR DESCRIPTION
**Pull request by logiclrd**
Added some text to the "Contributing the patches" section in `WELCOME.md` that provides a reference to the #git-devel IRC channel as a good place to find other contributors who can /allow you for GitGitGadget.

### CODE REVIEW

**Comment by dscho** [comment]
What about looking at other, recent GitGitGadget PRs where somebody `/allow`ed somebody? This would be the URL: https://github.com/gitgitgadget/git/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+%22%2Fallow%22

File: res/WELCOME.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ It is in general a good idea to await the automated test ("Checks") in this Pull
 
 Before you can contribute the patches, your GitHub username needs to be added to the list of permitted users. Any already-permitted user can do that, by adding a PR comment of the form `/allow <username>`. A good place to find other contributors is the channel [`#git-devel`](https://webchat.freenode.net/#git-devel) on the FreeNode IRC network.
 
-    <newcontributor> I've just created my first PR, could someone /allow me? https://github.com/gitgitgadget/git/pull/12345
+    <newcontributor> I've just created my first PR, could someone please /allow me? https://github.com/gitgitgadget/git/pull/12345
```
Comment (dscho) [inline res/WELCOME.md]: How about squashing that patch that adds the word "please" into the previous commit?

In the Git project, we prefer patch series without any "oops, I actually meant to do this, too" patches, i.e. we recommend to rewrite and polish the patches e.g. via interactive rebase until they shine, so to say.

**Comment by logiclrd** [comment]
I have incorporated that suggestion and squashed everything down to one commit. :-)

File: res/WELCOME.md [hunk 2]
```diff
@@ -15,7 +15,17 @@ It is in general a good idea to await the automated test ("Checks") in this Pull
 
 ## Contributing the patches
 
-Before you can contribute the patches, your GitHub username needs to be added to the list of permitted users. Any already-permitted user can do that, by adding a PR comment of the form `/allow <username>`.
+Before you can contribute the patches, your GitHub username needs to be added to the list of permitted users. Any already-permitted user can do that, by adding a PR comment of the form `/allow <username>`. A good way to find other contributors is to locate recent pull requests where someone has been `/allow`ed:
```
Comment (webstech) [inline res/WELCOME.md]: The default for `/allow` is to allow the PR author.  Should the `<username>` parameter be removed?

Just mentioning it while this is being updated.  Understand it is not really part of this change.

Comment (logiclrd) [reply res/WELCOME.md]: It would take me about 2 minutes to fold that into the commit, @dscho @nasamuffin opinions?

Comment (dscho) [reply res/WELCOME.md]: I'd love to have this change, but in a separate commit (although still in this PR).

Comment (logiclrd) [reply res/WELCOME.md]: Done. :-)

File: res/WELCOME.md [hunk 3]
```diff
@@ -15,7 +15,17 @@ It is in general a good idea to await the automated test ("Checks") in this Pull
 
 ## Contributing the patches
 
-Before you can contribute the patches, your GitHub username needs to be added to the list of permitted users. Any already-permitted user can do that, by adding a PR comment of the form `/allow <username>`.
+Before you can contribute the patches, your GitHub username needs to be added to the list of permitted users. Any already-permitted user can do that, by adding comment to your PR of the form `/allow`. A good way to find other contributors is to locate recent pull requests where someone has been `/allow`ed:
```
Comment (webstech) [inline res/WELCOME.md]: `by adding comment to your PR` should probably be `by adding a comment to your PR`.  Your rewording improves the original but the missing 'a' affects readability.  Sorry I didn't notice it earlier.

Comment (logiclrd) [reply res/WELCOME.md]: Whoops! Fixed.

**Comment by dscho** [comment]
Excellent! Thank you all!
